graylog2-server/src/main/java/org/graylog2/security/ldap/LdapConnector.java

```diff
@@ -251,6 +253,12 @@ public LdapEntry search(LdapNetworkConnection connection,
                     }
                 }
             }
+        } catch (Exception e) {
+            LOG.warn(
```

Comment:
There can only be 1 active system notification of a type, so this wouldn't swamp the administrator with notifications.